### ISSUE TITLE
Default Access-Point Credentials

### ISSUE THREAD

**Issue by markdeegan** [jasoneflood/jaymarine#6]
I guess we should probably decide on default WiFi access-point credentials for this.
Not urgent, but I have a feeling my mobile is going to become a roaming access-point.

**Comment by jasoneflood** [comment]
yeah - maybe keep it simple - jaymarine:jaymarine. The default is really a fallback anyway. I think we also need to consider the merits of a DHCP service to keep us away from IP hell with various dashboards. Not sure best way to do that, or if DHCP introduces a massive single point of failure.  I believe - but have not tested - that an ESP32 can work as a repeater/router. Getting that going could be a complete distraction in the short term.

**Comment by markdeegan** [comment]
Right so
jaymarine:jaymarine it is.
IF we are going to use DHCP (from a configurable DHCP server, such as RaspberryPi, then we can add our own settings to ber pushed by the DHCP server to a specific network, again, with a default setting for same).
